stack.yaml

```diff
@@ -0,0 +1,14 @@
+# This is the implicit global project's config file, which is only used when
+# 'stack' is run outside of a real project.  Settings here do _not_ act as
+# defaults for all projects.  To change stack's default settings, edit
+# '/Users/keagan/.stack/config.yaml' instead.
```

Comment:
I don't know much about stack, but I think this comment line needs revision. :)

Reply:
sure does